From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paolo Sabatino <paolo.sabatino@gmail.com>
Date: Thu, 24 Mar 2022 09:21:56 +0000
Subject: [ARCHEOLOGY] rockchip: fix kernel device tree in u-boot

> X-Git-Archeology: - Revision 2cbd35f3b2fbc90e4c93f3c1669fcd0f459d08a4: https://github.com/armbian/build/commit/2cbd35f3b2fbc90e4c93f3c1669fcd0f459d08a4
> X-Git-Archeology:   Date: Thu, 24 Mar 2022 09:21:56 +0000
> X-Git-Archeology:   From: Paolo Sabatino <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: rockchip: fix kernel device tree in u-boot
> X-Git-Archeology:
> X-Git-Archeology: - Revision 445062b8dc243c55381db7f5a579a08ca6a0d73a: https://github.com/armbian/build/commit/445062b8dc243c55381db7f5a579a08ca6a0d73a
> X-Git-Archeology:   Date: Thu, 24 Mar 2022 09:21:56 +0000
> X-Git-Archeology:   From: Paolo Sabatino <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: rockchip: advance tinkerboard u-boot to v2022.01
> X-Git-Archeology:
> X-Git-Archeology: - Revision c674bac80c15025081086ef481e31365dddc74c2: https://github.com/armbian/build/commit/c674bac80c15025081086ef481e31365dddc74c2
> X-Git-Archeology:   Date: Thu, 24 Mar 2022 09:21:56 +0000
> X-Git-Archeology:   From: Paolo Sabatino <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: rockchip: advance tinkerboard u-boot to v2021.01
> X-Git-Archeology:
> X-Git-Archeology: - Revision de26797423e22d58ec2882d7032c67f77196ecc5: https://github.com/armbian/build/commit/de26797423e22d58ec2882d7032c67f77196ecc5
> X-Git-Archeology:   Date: Sun, 06 Nov 2022 20:32:46 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move all legacy u-boot patches under one general legacy folder (#4386)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 97c34489831f2146940f52915428263b7edfcbe1: https://github.com/armbian/build/commit/97c34489831f2146940f52915428263b7edfcbe1
> X-Git-Archeology:   Date: Fri, 24 Mar 2023 23:13:42 +0100
> X-Git-Archeology:   From: Paolo Sabatino <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: rockchip: put all rockchip 32 bit into uboot/v2022.04 directory
> X-Git-Archeology:
---
 configs/tinker-rk3288_defconfig | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/configs/tinker-rk3288_defconfig b/configs/tinker-rk3288_defconfig
index 111111111111..222222222222 100644
--- a/configs/tinker-rk3288_defconfig
+++ b/configs/tinker-rk3288_defconfig
@@ -18,11 +18,12 @@ CONFIG_DEBUG_UART=y
 CONFIG_SYS_LOAD_ADDR=0x800800
 # CONFIG_ANDROID_BOOT_IMAGE is not set
 CONFIG_USE_PREBOOT=y
-CONFIG_DEFAULT_FDT_FILE="rk3288-tinker.dtb"
+CONFIG_DEFAULT_FDT_FILE="rk3288-tinker-s.dtb"
 CONFIG_SILENT_CONSOLE=y
 CONFIG_DISPLAY_BOARDINFO_LATE=y
 CONFIG_SPL_STACK_R=y
 CONFIG_SPL_STACK_R_MALLOC_SIMPLE_LEN=0x2000
+CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR=0x200
 CONFIG_SPL_I2C=y
 CONFIG_SPL_POWER=y
 CONFIG_CMD_GPIO=y
@@ -76,6 +77,7 @@ CONFIG_REGULATOR_RK8XX=y
 CONFIG_PWM_ROCKCHIP=y
 CONFIG_RAM=y
 CONFIG_SPL_RAM=y
+CONFIG_DM_RESET=y
 CONFIG_DEBUG_UART_SHIFT=2
 CONFIG_SYSRESET=y
 CONFIG_USB=y
@@ -86,6 +88,9 @@ CONFIG_USB_HOST_ETHER=y
 CONFIG_USB_ETHER_ASIX=y
 CONFIG_USB_ETHER_SMSC95XX=y
 CONFIG_USB_GADGET=y
+CONFIG_USB_GADGET_MANUFACTURER="ASUS"
+CONFIG_USB_GADGET_VENDOR_NUM=0x0b05
+CONFIG_USB_GADGET_PRODUCT_NUM=0x7820
 CONFIG_USB_GADGET_DWC2_OTG=y
 CONFIG_DM_VIDEO=y
 CONFIG_DISPLAY=y
-- 
Armbian

